src/quicr_client.cpp

```diff
@@ -86,19 +90,19 @@ QuicRClient::subscribe(std::shared_ptr<SubscriberDelegate> subscriber_delegate,
 }
 
 void
-QuicRClient::unsubscribe(const quicr::Namespace& quicr_namespace,
-                         const std::string& origin_url,
-                         const std::string& auth_token)
+Client::unsubscribe(const quicr::Namespace& quicr_namespace,
+                    const std::string& origin_url,
+                    const std::string& auth_token)
```

Comment:
Are these parameters really needed to unsubscribe?  It's not like you need authorization to leave like you do to join.

Reply:
No idea. But I'm avoiding crazy API changes as much as possible for now. Would love to address this in a follow up PR though.